### PR TITLE
ceph-volume-ansible-prs: create job template(s)

### PR DESCRIPTION
**Pull request by alfredodeza**
Signed-off-by: Alfredo Deza <adeza@redhat.com>

### CODE REVIEW

File: ceph-volume-ansible-prs/build/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+# the following two methods exist in scripts/build_utils.sh
+pkgs=( "tox" )
+install_python_packages "pkgs[@]"
+
+# trims leading slashes
+BRANCH=${GIT_BRANCH#*/}
```
Comment (andrewschoen) [inline ceph-volume-ansible-prs/build/build]: why not use the ``branch_slash_filter`` method?

File: ceph-volume-ansible-prs/build/build [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+SHA=$GIT_COMMIT
+
+
+cd src/ceph-volume/tests/functional && $VENV/tox -rv
```
Comment (andrewschoen) [inline ceph-volume-ansible-prs/build/build]: I don't think you want the ``&& $VENV/tox -rv`` here

File: ceph-volume-ansible-prs/build/build [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+CEPH_DEV_BRANCH=$BRANCH CEPH_DEV_SHA1=$SHA tox -ve $DISTRO-$SCENARIO
```
Comment (andrewschoen) [inline ceph-volume-ansible-prs/build/build]: add the ``-r`` flag here

File: ceph-volume-ansible-prs/build/teardown [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# There has to be a better way to do this than this script which just looks
+# for every Vagrantfile in scenarios and then just destroys whatever is left.
+
+cd $WORKSPACE/tests
```
Comment (andrewschoen) [inline ceph-volume-ansible-prs/build/teardown]: this path isn't correct now

Comment (alfredodeza) [inline ceph-volume-ansible-prs/build/teardown]: this doesn't matter really because it uses `find`, but I fixed this (and the rest) regardless

**Pull request merged**